project/AppDependencies.scala

```diff
@@ -7,14 +7,14 @@ object AppDependencies {
 
   private val compile = Seq(
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"                    % mongoVersion,
```

Comment:
I believe we are safe to update the Mongo dependencies to 2.1.0 as part of this.